src/util/newconfig/ConfigDescription.hpp

```diff
@@ -79,22 +78,42 @@ struct ClioConfigDescription {
      * @return An Error if generating markdown fails, otherwise nothing
      */
     [[nodiscard]] static std::expected<void, Error>
-    getMarkdown(std::string_view path)
+    generateConfigDescriptionToFile(std::filesystem::path path)
     {
         namespace fs = std::filesystem;
 
         // Construct the full file path
-        auto const fullFilePath = fmt::format("{}/{}", path, kCONFIG_DESCRIPTION_FILE_NAME);
+        fs::path const fullFilePath = path.string();
```

Comment:
I don't understand this change. You are given a path and then you copy it into another `path` object by first getting a string out of it. Does this differ from the original `path` you were given at all?

Reply:
fyi: what I meant previously by saying you should use `filesystem::path` to compose paths is that you could do something like 
```
auto const fullPath = path / kCONFIG_DESCRIPTION_FILE_NAME;
```
this composes the path correctly on all platforms while string manipulation using `{}/{}` would produce a broken path on say Windows (not that we care).